### PR TITLE
Hf history

### PR DESCRIPTION
**Pull request by terrikwak**
## Proposed changes

Hardfork page

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by terrikwak** [comment]
@kjhman21 @dcground @aidan-kwon  Please take a look

**Review by aidan-kwon — APPROVED** [review]
Great. I think it would be better if we have an additional page listing hard-fork items. Reference: https://ethereum.org/en/history/

**Comment by terrikwak** [comment]
> Great. I think it would be better if we have an additional page listing hard-fork items. Reference: https://ethereum.org/en/history/

Can you look at it again?? @aidan-kwon

**Comment by terrikwak** [comment]
@kjhman21 @dcground  please take a look

File: docs/klaytn-history/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Klaytn Hard Fork History
+
+This page shows all the major milestones, forks and updates to the Klaytn blockchain.
```
Comment (kjhman21) [inline docs/klaytn-history/README.md:3]:
```suggestion
This page shows all the hard forks to the Klaytn blockchain.
```

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -297,6 +297,7 @@
     * [Kaikas](toolkit/kaikas.md)
     * [Klaytn Wallet](toolkit/klaytn-wallet.md)
     * [Klaytnscope](toolkit/klaytnscope.md)
+* [Klaytn History](klaytn-history/README.md)
```
Comment (kjhman21) [inline docs/SUMMARY.md:300]:
```suggestion
* [Klaytn Hard Fork History](klaytn-history/README.md)
```

**Review by aidan-kwon — APPROVED** [review]
Thank you!

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**